CHANGELOG.md

```diff
@@ -4,13 +4,32 @@
 
 - Feature:
     - Support collecting and reporting logs to backend (#147)
+    - Support profiling Python method level performance (#127
     - Add a new `sw-python` CLI that enables agent non-intrusive integration (#156)
-
-- New plugins:
-    - Falcon Plugin (#146)
+    - Add exponential reconnection backoff strategy when OAP is down (#157)
+    - Suport ignoring traces by http method (#143)
+    - `NoopSpan` on queue full, propagation downstream (#141)
+    - Support agent namespace. (#126)
+    - Added `grpc.max_connection_age_grace_ms` (#130)
+    - Support secure connection option for GRPC and HTTP (#134)
+
+- Plugins:
+    - Add Falcon Plugin (#146)
+    - Update `sw_pymongo.py` to be compatible with cluster mode (#150)
+    - Add Python celery plugin (#125)
+    - Support tornado5+ and tornado6+ (#119)
```

Comment:
This was when the tornado plugin was needlessly split into a v5 and v6 version even though it was the same code, it was reverted.